Resources/public/js/views/dashboard/ez-dashboardblockasynchronousview.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockasynchronousview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the All Content Dashboard Block View class
+     *
+     * @module ez-dashboardblockallcontentview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_ROW = '.ez-block-row',
+        SELECTOR_CONTENT = '.ez-block-content',
+        EVENTS = {};
+
+    EVENTS[SELECTOR_ROW] = {tap: '_uiShowRowOptions'};
+
+    /**
+     * The asynchronous dashboard block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockAsynchronousView
+     * @constructor
+     * @extends eZ.DashboardBlockBaseView
+     */
+    Y.eZ.DashboardBlockAsynchronousView = Y.Base.create('dashboardBlockAsynchronousView', Y.eZ.DashboardBlockBaseView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._getContent;
+            this._watchAttribute = 'content';
+            /**
+             * Stores the click outside event handler
+             *
+             * @property _clickOutsideHandler
+             * @type {Object}
+             * @protected
+             */
+            this._clickOutsideHandler = null;
+            this.events = Y.merge(this.events, EVENTS);
+
+            this.get('container')
+                .addClass(this._generateViewClassName(Y.eZ.DashboardBlockBaseView.NAME))
+                .addClass(CLASS_LOADING);
+
+            this.after('contentChange', this._uiEndLoading);
+        },
+
+        /**
+         * Renders the dashboard block view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockAllContentView} the view itself
+         */
+        render: function () {
```

Comment:
actually, after thinking a bit about that, this base class should provide a `render` method because the rendering logic will be the same, the only difference will be how the `items` content is transformed to something the template can understand. So I would write the `render` method like:
```js
        render: function () {
            var items = this.get('items').map(function (item) {
                    return this._getTemplateItem(item);
                }, this);

            this.get('container').setHTML(this.template({
                items: items,
                loadingError: this.get('loadingError'),
            }));

            return this;
        },
```

and declare `_getTemplateItem` the same way as the method which fires the event to retrieve the data.
That way, the blocks using this class will just have to implement those 2 methods.